Define type ranges for application-specific parameters and extension headers (#1272)
moq-wg/moq-transport: draft-ietf-moq-transport.md

The subscription has Publisher Priorty 128 if this parameter is omitted.

### Application-specific Parameters {#application-specific-params}

> Applications may want to define parameters with semantics specific to that

Reviewer: ```suggestion
Applications can define parameters with semantics specific to that
```

> application. For this purpose, applications are free to use parameter types within

Reviewer: ```suggestion
application. For this purpose, applications can use parameter types within
```

the range 0xff0000 to 0xffffff, as those will not be used by any future versions of

> the MoQ specification.

Reviewer: ```suggestion
the MOQT specification.
```

This extension can be added by the Original Publisher, but MUST NOT be added by
relays. This extension MUST NOT be modified or removed.

## Application-specific extension headers

> Applications may want to define extension headers with semantics specific to that

Reviewer: ```suggestion
Applications can define extension headers with semantics specific to that
```

> application. For this purpose, applications are free to use extension header types

Reviewer: ```suggestion
application. For this purpose, applications can use extension header types
```

within the range 0xff0000 to 0xffffff, as those will not be used by any future

> versions of the MoQ specification.

Reviewer: ```suggestion
versions of the MOQT specification.
```
